doc/src/ForHosts.dox

```diff
@@ -111,19 +112,19 @@
  *
  * @subsection host_todo_required_publishing Required for Publishing
  *
- * - Ask the @ref manager if they support publishing relevant data via
+ * - Ask the @ref manager if they support publishing for the relevant
+ *   data by supplying its @ref trait_set to
  *   @ref openassetio.hostAPI.Manager.Manager.managementPolicy
- *   "Manager.managementPolicy" using a suitable set of @needsref traits
- *   and a @ref Context with `kWrite` access, and respect the returned
- *   flags.
+ *   "Manager.managementPolicy" with a @ref Context with `kWrite` access,
+ *   respecting the returned flags.
  *
  * - Follow the @ref openassetio.hostAPI.Manager.Manager.managementPolicy
  *   "policy", @ref preflight, resolve, write, @ref register process
  *   illustrated @ref example_publishing_a_file "here" whenever
  *   generating new data.
  *
  * - Include in your documentation, any scenarios in which entities are
- *   registered to references known to be of a different @ref Specification.
+ *   registered to references known to be of a different @ref trait_set.
```

Comment:
[consider] Perhaps worth also updating the example immediately below to use trait sets rather than specifications?

Reply:
A bit hard to read, to be honest. It's clear what you're trying to say, but I think some grammar fixes would be worthwhile.  Perhaps e.g.

> For example, an editorial application registering the traits of an ImageSpecification to a reference with the traits of a ShotSpecification indicates that the images should be published "under" that shot.

Reply:
Thats so much better! thanks! Added [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/249/commits/03bd975ad83ddbccb73de71e02856136cbd308c4).